app/Http/Controllers/CourseController.php

```diff
@@ -47,7 +58,7 @@ public function create()
 
 		$teachers = User::where('role', config(self::ROLES)[self::TEACHER])->select('id', 'name')->get();
 		$courses = Course::where('course_type', 2)->get();
-		return view('courses.create', compact(self::TEACHERS, 'courses'));
+		return view('courses.create', compact(self::TEACHERS, self::COURSES));
```

Comment:
Se me hace que estos de repeating and define a constant de CodeClimate los podemos ignorar. En mi opinión afectan severamente el readability y no proveen nada. voy a ver si los podemos quitar pero como estaba antes me parecía mucho mejor.